qiskit/circuit/bit.py

```diff
@@ -19,7 +19,13 @@
 
 
 class Bit:
-    """Implement a generic bit."""
+    """Implement a generic bit.
+
+    .. note::
+        This class should be instantiated directly, but just a super class
```

Comment:
Grammar nit, do you mean this?

```suggestion
        This class should not be instantiated directly. This is just a superclass
```

Reply:
ups.. sorry. you are right